frontend/__tests__/components/route-pages.spec.tsx

```diff
@@ -3,19 +3,19 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
 
-import { RouteHostname } from '../../public/components/routes';
+import { RouteLocation, RouteStatus, RouteWarnings } from '../../public/components/routes';
 import { K8sResourceKind } from '../../public/module/k8s';
 
-describe('RouteHostname', () => {
-  it('renders a https hostname link when TLS Settings', () => {
+describe('RouteLocation', () => {
```

Comment:
Prefer `RouteLocation.displayName` (need to add it to the component file as well) so that it stays consistent through refactors.

Reply:
Hi @alecmerdler, updated all to use `.displayName`